aptible/resource_log_drain_test.go

```diff
@@ -0,0 +1,72 @@
+package aptible
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"testing"
+
+	"github.com/aptible/go-deploy/aptible"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccResourceLogDrain_elasticsearch(t *testing.T) {
+	rHandle := acctest.RandString(10)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLogDrainDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAptibleLogDrainElastic(rHandle),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("aptible_app.test", "handle", rHandle),
+					resource.TestCheckResourceAttr("aptible_app.test", "account_id", strconv.Itoa(testAccountId)),
+					resource.TestCheckResourceAttr("aptible_app.test", "drain_type", "elasticsearch_database"),
+				),
+			}, {
+				ResourceName:      "aptible_app.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckLogDrainDestroy(s *terraform.State) error {
+	client := testAccProvider.Meta().(*aptible.Client)
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "aptible_app" {
+			continue
+		}
+
+		logDrainID, err := strconv.Atoi(rs.Primary.Attributes["log_drain_id"])
+		if err != nil {
+			return err
+		}
+
+		logDrain, err := client.GetLogDrain(int64(logDrainID))
+		log.Println("Deleted? ", logDrain.Deleted)
+		if !logDrain.Deleted {
+			return fmt.Errorf("app %v not removed", logDrainID)
+		}
+
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func testAccAptibleLogDrainElastic(handle string) string {
+	return fmt.Sprintf(`
+resource "aptible_app" "test" {
```

Comment:
```suggestion
resource "aptible_log_drain" "test" {
```
i assume this should be a log_drain right?